vnext/Microsoft.ReactNative/Views/ViewManager.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <JSValue.h>
+#include <cxxreact/CxxModule.h>
+
+namespace Microsoft::ReactNative {
+
+struct ShadowNode;
```

Comment:
>ShadowNode [](start = 7, length = 10)

just for my education: why declare instead of #include the file? Doesn't this force the consumers to include ShadoNode? Or are there 2 versions?

Reply:
You can include this file without having the final definition of ShadowNode, only pointers to it are used in this header.  So consumers that are not actually using ShadowNode can save the compile time of needing to compile ShadowNode.h if they dont need it.